models/user.js

```diff
@@ -0,0 +1,48 @@
+'use strict'
+
+const  { Model } = require('objection')
+const Knex = require("knex")
+
+// Initialize knex.
+const knex = Knex({
+    client: 'sqlite3',
+    useNullAsDefault: true,
+    connection: {
+        filename: 'one-sg.db3'
+    }
+});
+
+Model.knex(knex)
```

Comment:
Since I'm looking at this PR anyway, this probably doesn't belong here and belongs in it's own initialization file

Reply:
Could we also move this `models` folder inside a parent `src` folder. If the project structure in the README doesn't mention a `src` folder - let's raise this up on the slack channel? (you'll need to read up a bit on folder hierarchy conventions to justify why we should have a one :) )

Reply:
noted on this